src/styles.scss

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+ $dds-prefix: 'dds' !default;
+ $prefix: 'bx' !default;
+
+#masthead-banner {
+  text-decoration: none;
+  color: #ffffff;
+  background: #262626;
+  border-top: 3px solid #262626;
+  border-left: 3px solid #262626;
+  border-right: 3px solid #262626;
+  border-bottom: 3px solid #00bab6;
+  box-sizing: border-box;
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
+  #masthead-banner-content {
+    box-sizing: border-box;
+  }
+
+  &:hover {
+    background: #161616;
+    border-color: transparent;
+    border-bottom: 3px solid #00bab6;
+  }
+
+  &:focus {
+    border: 3px solid #00bab6;
+    outline: none;
+
+    #masthead-banner-content {
+      outline: 1px solid #ffffff;
+    }
+  }
+
+  .masthead-banner--align-center {
+    align-items: center;
+  }
+
+  .masthead-banner--space-between {
+    justify-content: space-between;
+  }
+
+  .masthead-banner--flex {
+    display: flex;
+  }
+
+  .masthead-banner__icon {
+    margin: 0 24px;
+  }
+}
+
+#masthead-banner-desktop {
+  height: 75px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#masthead-banner-mobile {
+  height: 75px;
+}
+
+@media (max-width: 659px) {
+  #masthead-banner-desktop {
+    display: none;
+  }
+
+  #masthead-banner-mobile {
+    display: flex;
+  }
+}
+
+@media (min-width: 660px) {
+  #masthead-banner-desktop {
+    display: flex;
+  }
+
+  #masthead-banner-mobile {
+    display: none;
+  }
+}
+
+// THINK BANNER 2020
+#think-banner-link {
+  text-decoration: none;
+  background: #f4f5f6;
+  color: #000000;
```

Comment:
Wondering if we can use carbon tokens? (`$ui-background`)

Reply:
The reason I have it set up the way it is is to keep it as aligned with the v18 version as possible, which means sharing all styling. Keeping that in mind, I think it's safer to match the css exactly where possible, at least until we fully deprecate the v18 version.

Reply:
that makes sense, thanks for explaining!